multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -26,6 +26,7 @@
 from packaging import version
 from torch import nn
 
+import autogluon.multimodal.utils.object_detection
```

Comment:
This import style is different from others. Could we do `from .utils import convert_data_to_df`? Maybe also consider changing `convert_data_to_df` to `object_detection_data_to_df` to let the name more specific.

Reply:
Yea. This was because of one of the previous conversation with xingjian where he mentioned to use this kind of import instead of from `.utils import convert_data_to_df`. I can change it now